README.md

```diff
@@ -48,8 +48,8 @@ Install the following tools, if you don't have them already:
   and setup might not work if you use another shell.
 
 - **GNU Make**. 
-  On Windows the easiest way to install Make is `choco install make`. 
-  Other options include using a [subsystem][wsl].
+  On Windows the easiest way to install Make is `choco install make` using 
+  command prompt or powershell as admin. Other options include using a [subsystem][wsl].
```

Comment:
```suggestion
  On Windows the easiest way to install Make is `choco install make`
  using command prompt or powershell as as admin. 
  Other options include using a [subsystem][wsl].
```

Reply:
No problem. Was not sure about the spacing of the sentences. Thank you for assisting!